lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -176,7 +176,7 @@ describe('ShareDialogComponent', () => {
 
         fixture.detectChanges();
 
-        fixture.nativeElement.querySelector('.mat-slide-toggle label')
+        fixture.nativeElement.querySelector('[data-automation-id="adf-share-toggle"] label')
```

Comment:
When you address my my two above comments about creating functions which you use in each place then you can notice that both functions will use following data-automation-id: [data-automation-id="adf-share-toggle"]. Please move that id to some constant so both functions will use the same id for share toggle